core/commands/publish.go

```diff
@@ -89,7 +91,8 @@ Publish an <ipfs-path> to another public key (not implemented):
 		}
 
 		// TODO n.Keychain.Get(name).PrivKey
-		output, err := publish(n, n.PrivateKey, p)
+		// TODO(cryptix): is req.Context().Context a child of n.Context()?
```

Comment:
apparently `req.Context().Context()` is not yet a child of `n.Context()`: https://github.com/ipfs/go-ipfs/blob/master/commands/http/handler.go#L52-L54 -- it should be.